File: src/main/java/it/nanowar/ofsteel/helloworld/HelloWorldMainLauncherClass.java
package it.nanowar.ofsteel.helloworld;

public class HelloWorldMainLauncherClass {
	
	public HelloWorldMainLauncherClass(int foo) {
		
		super();
		this.foo=foo;
	}
	
	private Integer foo=0;
	
	public void songRefrain() {
		
		for (int i=0; i<foo; i++) {
			System.out.println("Hello World!");
			
		}
		
		int pippo=0;
		while (pippo<foo) {
			System.out.println("Hello World!");
			pippo++;
		}
	}
	
	public static void main (String [] args) {
		System.out.println("Hello World Programmer Start");
		HelloWorldMainLauncherClass tizio = new HelloWorldMainLauncherClass(2);
		tizio.songRefrain();
		System.out.println("Program Finished!");
		tizio.falseMethod();
	}
	
	/*
	 * If I may introduce a bug, the JVM will manage it for me
	 * ensuring both security and portability
	 * Then I'll write my code once, and run it everywhere!
	 * With Static and Strong Typing, will let my programs be type safe!
	 */
	
	public void falseMethod() {
		boolean metal=false;
		if(metal==false) {
			String joeyDeCaio=null;
			joeyDeCaio.length();
		}
	}
}

File: src/main/entity/it/nanowar/ofsteel/helloworld/HelloWorldMainLauncherClass.cpp


#include "HelloWorldMainLauncherClass.h"
#include "iostream"
#include <cstddef>
#include <cstring>

HelloWorldMainLauncherClass::HelloWorldMainLauncherClass(int foo)
: foo(foo) {

}

void HelloWorldMainLauncherClass::songRefrain(){
	for(int i = 0; i < foo; i++) {
		std::cout << "Hello World!" << std::endl;
	}

	int pippo = 0;
	while(pippo <foo){
		std::cout << "Hello World!" << std::endl;
		pippo++;
	}
}

void HelloWorldMainLauncherClass::falseMethod(){
	bool metal = false;
	if(metal == false){
		char* joeyDeCaio = NULL;
		strcpy(joeyDeCaio, "valhalleluja");
	}
}
